usaspending_api/etl/management/commands/load_table_to_delta.py

```diff
@@ -109,11 +109,11 @@
         "swap_schema": None,
         "partition_column": "id",
         "partition_column_type": "numeric",
-        "is_partition_column_unique": True,
-        "delta_table_create_sql": recipient_profile_sql_string,
+        "delta_table_create_sql": recipient_profile_create_sql_string,
+        "is_partition_column_unique": False,
```

Comment:
Shouldn't this still be `True`?

Reply:
No, recipient_profile has `recipient_hashes` for potentially all levels of any recipient hash - i.e., `recipient_hash-C` `recipient_hash-R`, `recipient_hash-P`

Reply:
The partition_column in this TABLE_SPEC is the `id` though not the hash

Reply:
oh fair, fixed